From ee0e1b5ffbf7c155b47d44ea86f11bfbbd0536d0 Mon Sep 17 00:00:00 2001
From: Shreenidhi Shedi <sshedi@vmware.com>
Date: Mon, 28 Mar 2022 20:42:43 +0530
Subject: [PATCH] system logging (rebased for 43.6)

Upstream logging.fc already lists /run paths; keep Photon /var/opt/audit only.

Signed-off-by: Shreenidhi Shedi <sshedi@vmware.com>
---
 policy/modules/system/logging.fc | 4 +++++
 1 file changed, 4 insertions(+)

diff --git a/policy/modules/system/logging.fc b/policy/modules/system/logging.fc
index 8dd07cd..16a0400 100644
--- a/policy/modules/system/logging.fc
+++ b/policy/modules/system/logging.fc
@@ -59,6 +59,10 @@
 /var/log/audit\.log.*	--	gen_context(system_u:object_r:auditd_log_t,mls_systemhigh)
 ')
 
+ifdef(`distro_photon',`
+/var/opt/audit(/.*)?		gen_context(system_u:object_r:auditd_log_t,mls_systemhigh)
+')
+
 ifdef(`distro_redhat',`
 /var/named/chroot/var/log -d	gen_context(system_u:object_r:var_log_t,s0)
 /var/named/chroot/dev/log -s	gen_context(system_u:object_r:devlog_t,s0)
